exe/dumple

```diff
@@ -62,7 +65,7 @@ begin
     dump_path = "#{dump_dir}/#{config['database']}_#{Time.now.strftime("%Y%m%d_%H%M%S")}.dump"
   end
 
-  puts "> Dumping database for \"#{environment}\" environment ..."
+  puts "> Dumping \"#{config['database'] + ' '}\"database for \"#{environment}\" environment ..."
```

Comment:
Ok, this was ambiguous, but for consistency with the rest of Geordi the value of the `database` variable should be used:

```suggestion
  given_database = database ? %(#{database} ) : ""
  puts "> Dumping #{given_database}database for \"#{environment}\" environment ..."
```